frontend/screens/SignupScreen/index.js

```diff
@@ -54,16 +56,34 @@ function index() {
     if (!flag) {
       console.log('sending account to database');
       setLoadingIcon(true);
-      postUser(username, email, phone, password, displayName);
+      postUser(username, email, phone, password, displayName)
+        .then((response) => {
+          console.log(response);
+          if (response.status >= 200 && response.status < 300) {
```

Comment:
I think checking the status of the request should be done in the controller. All that the view should is just receive the data and decide how to display it.